r/rsparkling/R/package.R

```diff
@@ -1,13 +1,24 @@
-
 #' @import sparklyr h2o utils
 NULL
 
 # define required spark packages
 spark_dependencies <- function(spark_version, scala_version, ...) {
+  sw_version = getOption("rsparkling.sparklingwater.version", default = "1.6.7")
```

Comment:
@jangorecki But it still hardcodes the versions in the mapping - with each new version, you would need to change the mapping and thus RSparkling. From my point of view, explicit setting is better.

Regarding 
> We still need to hardcode mapping of spark-sparkling versions.
 sparkling-water version tells us for which spark and scala it's being targeted, so this may not be necessary too